traits/trait_dict_object.py

```diff
@@ -61,6 +61,12 @@ def __init__(self, added=None, changed=None, removed=None):
             removed = {}
         self.removed = removed
 
+    def __repr__(self):
+        outstr = "TraitDictEvent(added={}, changed={}, removed={})".format(
```

Comment:
We probably want to build our `repr` out of the `repr` or the various pieces, instead of the `str` of those pieces. An easy way to do that is to use `{!r}` instead of `{}` for each of these placeholders. (See docs [here](https://docs.python.org/3/library/string.html?highlight=formatting#format-string-syntax) for a description of `{!r}`, if you haven't encountered it before.)

As it turns out, in this case I think it makes no difference, because the `str` and `repr` of a `dict` are identical. But it's good practice to do it anyway.